requirements.txt

```diff
@@ -1,7 +1,5 @@
 # In-house dependencies
-leaf_common==1.2.21
-leaf_server_common==0.1.18
```

Comment:
No need to specify `leaf_common` and `leaf_server_common` anymore: they come with `neuro-san`.